go-controller/pkg/libovsdbops/switch.go

```diff
@@ -119,6 +119,24 @@ func FindAllNodeLocalSwitches(nbClient libovsdbclient.Client) ([]nbdb.LogicalSwi
 	return switches, nil
 }
 
+// FindSwitchByName finds switch with provided name. If more than one is found, it will error.
+func FindSwitchByName(nbClient libovsdbclient.Client, name string) (*nbdb.LogicalSwitch, error) {
```

Comment:
nit: i would have like to have seen findSwitch turn into FindSwitch and just return the entire switch it finds, rather than making 2 functions that essentially do the same functionality. We can clean it up later though.

Reply:
ack. sounds reasonable.